packages/duckietown_demos_dbv2/launch/lane_following.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+	<arg name="veh" default="$(env VEHICLE_NAME)"/>
+	<arg name="ai_trafo_mode" default="cb" doc="'cb' for colo balance only; 'both' for color balance and linear trafo"/>
+	<arg name="ai_interval" default="5" doc="interval with which the linear trafo gets updated. color balance is performed every second."/>
```

Comment:
So those are here, are they not needed for the indefinite navigation demo as well then ?

Reply:
I mean ai_interval, ai_trafo_mode and such

Reply:
I don't know, I copied this launch file over from DBV1.